pkg/stanza/operator/helper/test_common.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package helper // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/helper"
```

Comment:
This file name should end with `_test.go` so that it is properly identified as test code.

Reply:
how would do so and still allow import to input/syslog? i know https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/operator/parser/syslog/data.go is being name the way it is to be used in input/syslog and parser/syslog.